README.md

```diff
@@ -21,9 +21,9 @@ The library is available from Pypi and can be installed with pip:
 
 ## Developing
 
-Install requirements:
+Install dependencies:
 
-> pip install -r requirements/core.txt -r requirements/dev.txt
+> make install
```

Comment:
Is there any ability to install a non-dev version of master using pip without picking up the dev dependencies?  Is there any utility in doing that?

Reply:
Yeah, when building the package (in order to publish to PyPI) we'll just install what's in core. I think that will be in a separate GH Action workflow specification. So for this particular workflow I think this is correct since it's running the linter and unit tests. At that time I'd either modify the install target or add a new one / some combo thereof.

Reply:
To expand on what I wrote, I think the install target installing both core and dev is fine. My hope (?) is that we can move to something like poetry which, like npm, handle dev and runtime dependencies, and can install with or without dev dependencies.